packages/datadog-instrumentations/src/helpers/register.js

```diff
@@ -75,10 +69,6 @@ for (const packageName of names) {
   if (hook !== null && typeof hook === 'object') {
     if (hook.serverless === false && isInServerlessEnvironment()) continue
 
-    // some integrations are disabled by default, but can be enabled by setting
-    // the DD_TRACE_<INTEGRATION>_ENABLED environment variable to true
-    if (hook.disabled && !reenabledInstrumentations.has(normalizedPackageName)) continue
```

Comment:
I've not been able to find any hooks that are disabled by default currently. Am I correct in assuming that this was just an option, but that we currently didn't use it?

Does this PR also remove support for the `disabled` property on hooks?